Utils/insertBoxesToDB.js

```diff
@@ -6,7 +6,7 @@
 const importXlsxFile = require('./importXlsxFile');
 const insertBoxes = require('../server/database/queries/insertBoxes');
 
-const path = '/home/ray261/Desktop/BlackSheet/MINI_BOXES.xlsx';
+const path = process.argv[2];
```

Comment:
Continuing the conversation from yesterday, your client would probably not have a developer on hand so this is still bad UX, your app is here to help a party access their voters in an easier way, I suggest adding a page or an input box where the user would input a path to the file where you can upload the voters to your database.
This in this form is not ideal, I will merge it if you guys are happy with the user journey as it is now! :maple_leaf:

Reply:
We will certainly take this into consideration for improving the app 🙏